src/test/java/seedu/address/model/util/DateTimeTest.java

```diff
@@ -3,81 +3,70 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-import java.util.Optional;
-
 import org.junit.jupiter.api.Test;
 
 class DateTimeTest {
 
     public static final String VALID_SAMPLE_A = "30-11-2020 10:00";
-    public static final String VALID_DATE_A = "30-11-2020";
-    public static final String VALID_SAMPLE_B = "31-11-2020 23:00";
-    public static final String VALID_DATE_B = "30-02-2020";
-    public static final String VALID_DATE_C = "29-02-2019";
-    public static final String TIME_A = "23:00";
+    public static final String VALID_SAMPLE_B = "28-02-2020 10:00";
+    public static final String VALID_SAMPLE_C = "28-02-2020 10:01";
```

Comment:
more valid inputs: 
-29th February for leap years

more invalid inputs:
-with seconds

vary the years to see how far back / into the future can be accepted / rejected
feel free to think of more test cases